app/mobile/src/README.md

```diff
@@ -0,0 +1,24 @@
+# Mobile app development
+
+We are building the mobile app using react native & expo.
+
+To run locally for development purposes, you will need an Android or iOS phone
+with the Expo app installed. The app can also be run in an emulator but emulator
+setup is currently outside the scope of this document.
+
```

Comment:
> The app can also be run in an emulator but emulator setup is currently outside the scope of this document.

Are there even some hacky instructions for this you might be able to share on Slack for the time being, until this process is ironed out more?

Reply:
added links to emulator/simulator to the readme.